system/jlib/jptree.cpp

```diff
@@ -9014,16 +9024,12 @@ jlib_decl IPropertyTree * loadConfiguration(IPropertyTree *componentDefault, con
      * on-demand, which means this mechanism is sufficient to cover most cases.
      */
 
-#if defined(__linux__) && defined(_CONTAINERIZED)
+#if defined(__linux__)
     /* In bare-metal - there is no auto process/component restart mechanism, so everything would need to be
```

Comment:
Maybe remove comment now ?

Reply:
yes, and/or comment above it needs updaing. I'll do that.